src/main/minio.js

```diff
@@ -1387,11 +1389,7 @@ export class Client {
       // is not expensive as the socket will not have any data.
       response.on('data', ()=>{})
 
-      var result = {
-        size: +response.headers['content-length'],
-        contentType: response.headers['content-type'],
-        lastModified: new Date(response.headers['last-modified'])
-      }
+      let result = Object.assign({}, {size:+response.headers['content-length'] }, extractMetadata(response.headers),{lastModified: new Date(response.headers['last-modified'])})
```

Comment:
I changed it so that result is basically the size and the date + the rest of the metadata. Should be ok, but just wanted to flag it if there was an error/bad logic